### ISSUE TITLE
Create or point to pre-exercise ROOT

### ISSUE THREAD

**Issue by caredg** [cms-opendata-workshop/2020-09-30-cms-open-data-workshop-for-theorists#16]
Create some instructions on how to access ROOT and how to use it.  This is linked to cernopendata/cms-opendata-guide#23

**Comment by caredg** [comment]
I am a bit worried about the HTT example needing ROOT with version > 6.16.  There is a nice [root intro](https://cmsdas.github.io/root-short-exercise/) in the cmsdas repository already in place.  I am not sure how we will get v6.16 or greater in an easy way? Does @someone have a clear view on this?

**Comment by caredg** [comment]
Ok, my bad.  I have tested (I forgot if I already did before, I guess I did) these [instructions](https://github.com/cms-opendata-analyses/HiggsTauTauNanoAODOutreachAnalysis/tree/2012#how-to-run-this) and the root setup from cvmfs works just fine (in a container with access to host cvmfs client).  So I presume it is the same in the VM.  

So, I believe we could very well use (actually directly pointing to some episodes) some sections in [this](https://cmsdas.github.io/root-short-exercise/) lesson from cmsdas.  The RDataFrame episode, for example (the one I checked), even uses an open data file, so it works beautifully.
